DittoToolsApp/DittoToolsApp.xcodeproj/project.pbxproj

```diff
@@ -746,12 +753,12 @@
 /* End XCConfigurationList section */
 
 /* Begin XCRemoteSwiftPackageReference section */
-		0E523F442B59B1A0007D6665 /* XCRemoteSwiftPackageReference "DittoSwiftTools" */ = {
+		14E35DB22B7F356A0018EC3B /* XCRemoteSwiftPackageReference "DittoSwiftPackage" */ = {
 			isa = XCRemoteSwiftPackageReference;
-			repositoryURL = "https://github.com/getditto/DittoSwiftTools";
+			repositoryURL = "https://github.com/getditto/DittoSwiftPackage";
 			requirement = {
 				kind = upToNextMajorVersion;
-				minimumVersion = 4.0.0;
+				minimumVersion = 4.5.3;
```

Comment:
This feels like it changes how the dependencies work for the project/package. I think today the `DittoToolsApp` Xcode project pulls in the `DittoSwiftTools` SPM package from this same repo for dogfooding purposes, although doing that via the remote URL is a bit wonky - probably that should pull it in with a local path reference, and the `Package.swift` at the root of this repo for the `DittoSwiftTools` package should have this update to >=4.5.3?

Reply:
Got it, I definitely misunderstood the dependencies here.